docs/pages/database-access/rbac.mdx

```diff
@@ -138,3 +138,213 @@ spec:
+    # match adds objects to be imported; it cannot be empty.
+    match:
+      # list of all table names
+      table_names:
+      - '*'
+  # Additional mappings can be added here.
+  # - add_labels: ...
+version: v1
+```
+
+This rule will import all objects and label them by their inherent properties using the template syntax.
+
+### Customizing the default import rule
+
+You can modify the default `db_object_import_rule` as you would any other
+Teleport resource.
+
+For instance, consider the following rule designed to designate particular
+tables as accessible to developers, either in a read-only or read-write
+capacity:
+
+```yaml
+kind: db_object_import_rule
+metadata:
+  name: ownership_nonprod
+spec:
+  priority: 100
+  database_labels:
+  # Affect `dev` and `staging` environments.
+  # Prod environment may have a different rule.
+  - name: 'env'
+    values:
+    - 'staging'
+    - 'dev'
+    - 'prod'
+  mappings:
+  # Apply project label
+  - add_labels:
+      project: horizon
+    # match section is mandatory and must contain at least one non-empty subsection
+    match:
+      table_names:
+      - '*'
+    # scope is the optional section which enables further filtering of objects by database and schema names. When omitted, this filtering is disabled.
+    scope:
+      database_names:
+      - 'horizon'
+      - 'horizon_v2'
+      schema_names:
+      - 'application'
+      - 'data_import'
+  # Add `dept: hr` label for respective tables.
+  - add_labels:
+      dept: hr
+    match:
+      table_names:
+      - '*'
+    scope:
+      schema_names:
+      - 'recruitment'
+      - 'salaries'
+      - 'pto'
+      - 'hr_scratchpad'
+version: v1
+```
+
+### Database admin user
+
+A database admin user is responsible for granting permissions to end users. You
+must specify a database admin user before using database object import rules. To
+specify a database admin user, add the following to a role that restricts
+database object permissions:
+
+```yaml
+spec:
+  # ...
+  admin_user: "teleport-admin"
+  # ...
+```
+
+In this case, the Teleport Database Service expects to activate a user called
+`teleport-admin` in order to execute object import rules.  Ensure that the admin
+user possesses the necessary permissions to manage users in your database,
+otherwise object import rules might fail, depending on how you have configured
+the database:
+
+```code
+$ tsh db connect postgres-db --db-name postgres --db-user teleport-user
+psql: error: connection to server at "localhost" (::1), port 50800 failed: Connection refused
+	Is the server running on that host and accepting TCP/IP connections?
+connection to server at "localhost" (127.0.0.1), port 50800 failed: your Teleport role requires automatic database user provisioning but an attempt to activate database user "teleport-user" failed due to the following error: ERROR: permission denied for table pg_subscription (SQLSTATE 42501)
+ERROR: exit status 2
+```
```

Comment:
`admin_user` is a property of the database spec, which may be missed by the user without the type of object apparent from the `yaml` listing.